draft-regext-rdap-x-media-type.md

```diff
@@ -113,43 +113,43 @@ they do not support, servers SHOULD NOT list extensions in the RDAP-X media type
 
 Nothing in this specification sidesteps or obviates the HTTP content negotiation defined
-in [@!RFC9110] for RDAP. Specifically, if a client gives RDAP-X a lower qvalue than
+in [@!RFC9110] for RDAP. Specifically, if a client gives RDAP-X a lower q value than
 any other media type, that is a signal not to use RDAP-X.
 
 Likewise, nothing in this specification sidesteps or obviates the HTTP caching mechanisms
-defined in [@!RFC9110]. Further advice on the `vary` header may be found in (#vary_header).
+defined in [@!RFC9110]. Further advice on the `vary` header can be found in (#vary_header).
 
-Some RDAP extensions, such as [@?I-D.ietf-regext-rdap-openid], have other protocol elements
+Some RDAP extensions, such as [@?RFC9560], have other protocol elements
 passed from the client to the server, and the presence of these protocol elements may be
 used by servers to determine a client's capability to handle the RDAP extension. This specification
-does not require the usage of those extensions identifiers in the extensions parameter,
-though clients SHOULD list the extension identifier in the extensions parameter when using
+does not require the usage of those extension identifiers in the "extensions" parameter,
+though clients SHOULD list the extension identifier in the "extensions" parameter when using
 other protocol elements of those extensions. Servers SHOULD NOT require the usage of extension
-identifiers in the extensions paramater when other extension protocol elements are used.
+identifiers in the "extensions" parameter when other extension protocol elements are used.
 
 # Usage in RDAP Links {#links}
 
 [@!RFC9083, section 4.2] defines a link structure used in RDAP.
 
     {
-      "value" : "https://example.com/context_uri",
-      "rel" : "self",
-      "href" : "https://example.com/target_uri",
-      "hreflang" : [ "en", "ch" ],
-      "title" : "title",
-      "media" : "screen",
-      "type" : "application/json"
+      "value": "https://example.com/context_uri",
+      "rel": "self",
+      "href": "https://example.com/target_uri",
+      "hreflang": [ "en", "ch" ],
+      "title": "title",
+      "media": "screen",
+      "type": "application/json"
     }
 
 The type attribute signals to a client the expected media type of the resource
 referenced in the href attribute, and some clients use this information to determine
-if the URI in the href attribute should be dereferenced.
+if the URI in the href attribute should be de-referenced.
 
 Servers MAY use the RDAP-X media type in the type attribute if a client
 has negotiated content with the server using the RDAP-X media type, 
 the resource referenced by the URI matches the RDAP-X media type, and
 the resource referenced by the URI is served by a server compliant with this specification.
-Otherwise, use of the `application/rdap+json` media type is RECOMMENDED when the URI
+Otherwise, use of the 'application/rdap+json' media type is RECOMMENDED when the URI
```

Comment:
The backtick usage is intentional as it tell markdown to use code style formatting such as monospace fonts.

Such as application/rdap+json vs `application/rdap+json`